source/installer.py

```diff
@@ -362,8 +363,7 @@ def tryRemoveFile(path,numRetries=6,retryInterval=0.5,rebootOK=False):
 		time.sleep(retryInterval)
 	if rebootOK:
 		log.debugWarning("Failed to delete file %s, marking for delete on reboot"%tempPath)
-		MoveFileEx=windll.kernel32.MoveFileExW
-		MoveFileEx("\\\\?\\"+tempPath,None,4)
+		winKernel.moveFileEx("\\\\?\\"+tempPath,None,winKernel.MOVEFILE_DELAY_UNTIL_REBOOT)

```

Comment:
Note that this could now raise an error while it couldn't before. It might make sense to put this in a try/except block, log an error when there is a failure but return regardless. E.g:

```suggestion

		try:
			winKernel.moveFileEx("\\\\?\\"+tempPath,None,winKernel.MOVEFILE_DELAY_UNTIL_REBOOT)
		except WindowsError:
			log.debugWarning("Failed to mark file %s for delete on reboot"%tempPath, exc_info=True)
```